.zenodo.json

```
@@ -0,0 +1,162 @@
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Blelly, Aurore"
+      },
+      {
+        "affiliation": "LIDYL, CEA-Universit\u00e9 Paris-Saclay, CEA Saclay",
+        "name": "Fedeli, Luca"
+      },
+      {
+        "affiliation": "SLAC National Accelerator Laboratory",
+        "name": "Ge, Lixin"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Gott, Kevin"
+      },
+      {
+        "affiliation": "Lawrence Livermore National Laboratory",
+        "name": "Harrison, Cyrus"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Huebl, Axel",
+        "orcid": "0000-0003-1943-7141"
+      },
+      {
+        "affiliation": "Lawrence Livermore National Laboratory",
+        "name": "Grote, David"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Gu, Junmin",
+        "orcid": "0000-0002-1521-8534"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Jambunathan, Revathi"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory, now DESY",
+        "name": "Th\u00e9venet, Maxence"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Richardson, Glenn"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Shapoval, Olga"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Lehe, R\u00e9mi"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Loring, Burlen"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Myers, Andrew"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Rowan, Michael E."
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Yang, Eloise"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Zhang, Weiqun"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Zhao, Yinjian"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Zoni, Edoardo"
+      },
+      {
+        "affiliation": "LIDYL, CEA-Universit\u00e9 Paris-Saclay, CEA Saclay",
+        "name": "Zaim, Ne\u00efl"
+      }
+    ],
+    "contributors": [
+      {
+        "affiliation": "Intel",
+        "type": "Other",
+        "name": "Cohn, Robert"
+      },
+      {
+        "affiliation": "Bloomberg LP",
+        "type": "Other",
+        "name": "Levine, Michael"
+      },
+      {
+        "type": "Other",
+        "name": "Liu, Bin"
+      },
+      {
+        "type": "Other",
+        "name": "nckwcq"
```

@nckwcq what is your real name, affiliation and ORCiD? :)
```suggestion
        "name": "nckwcq",
       "orcid": "..."
```